src/parser/parser_support.c

```diff
@@ -10290,6 +10293,31 @@ pt_set_user_specified_name (PARSER_CONTEXT * parser, PT_NODE * node, void *arg,
   user_specified_name = pt_append_string (parser, downcase_resolved_name, ".");
   user_specified_name = pt_append_string (parser, user_specified_name, original_name);
 
+  name_size = (int) strlen (user_specified_name);
+  if (LANG_SYS_CODESET == INTL_CODESET_UTF8
+      && unicode_string_need_compose (user_specified_name, name_size, &composed_size, lang_get_generic_unicode_norm ()))
+    {
+      char *composed = NULL;
+      bool is_composed = false;
+
+      composed = STATIC_CAST (char *, (parser_allocate_string_buffer (parser, composed_size + 1, sizeof (char))));
+      if (composed == NULL)
+	{
+	  PT_ERRORf (parser, node, "cannot alloc %d bytes", composed_size + 1);
+	  return NULL;
+	}
+
+      unicode_compose_string (user_specified_name, name_size, composed, &composed_size, &is_composed,
+			      lang_get_generic_unicode_norm ());
```

Comment:
no necessary to check "is_composed" is true or false ?

Reply:
The same code is being used elsewhere.
Maybe check is_composed for stability?

For consistency with other code, I'll keep the check for is_composed.

Reply:
After analyzing with @ctshim, I found that this part of the code was unnecessary.
So, the code that normalizes Unicode has been removed here.